jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/servlets/SPARQL_Update.java

```diff
@@ -83,7 +83,7 @@ public void execOptions(HttpAction action) {
 
     @Override
     public void execGet(HttpAction action) {
-        ServletOps.errorMethodNotAllowed(HttpNames.METHOD_GET, "GET not support for SPARQL Update. Use POST or PATCH");
+        ServletOps.errorMethodNotAllowed(HttpNames.METHOD_GET, "SPARQL Update is not supported with GET. Use POST instead");
```

Comment:
```suggestion
        ServletOps.errorMethodNotAllowed(HttpNames.METHOD_GET, "SPARQL Update is not supported with GET. Use POST or PATCH instead");
```
I think it's better to leave mention of PATCH in. It is supported.

Yes, the general uptake of PATCH on the web does not seem huge, which is a pity; it was "only" standardised [March 2010](https://datatracker.ietf.org/doc/html/rfc5789).

PATCH is a good match for SPARQL Update.

Reply:
When I am trying to update my instance of Fuseki 4.8.0 with PATCH, I am getting the following error: "HTTP method not allowed: SPARQL Update : use POST" triggered by https://github.com/apache/jena/blob/f540f6df8b5492e5f052a1679d6e46b95fce744f/jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/servlets/SPARQL_Update.java#L128

Reply:
Oh. Then that's a bug because further up it does allow PATCH!

https://github.com/apache/jena/blob/ba2cdb6211f60d7725d779a8556266a8c0b2d0c5/jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/servlets/SPARQL_Update.java#L80

https://github.com/apache/jena/blob/ba2cdb6211f60d7725d779a8556266a8c0b2d0c5/jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/servlets/SPARQL_Update.java#L95

Could you add the fix to the PR please?